rails/commands-pattern/app/controllers/users_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class UsersController < ApplicationController
+
+  def create
+    authorize User
+
+    Users::Create.new(user_params).call.then do |result|
```

Comment:
I'm not convinced about using `then`. There are two problems:
1. Node developers associate `then` to some kind of asynchronous execution. This is a ruby problem.
2. The code is IMO unnecessarily complex. I prefer to read:

```ruby
result = Users::Create.new(user_params).call
if result.ok
...
```